lib/utilities.js

```diff
@@ -61,7 +62,52 @@ function invertObject(obj, override) {
   return result;
 }
 
+/**
+ * Check if a given tag is a custom component.
+ *
+ * @see https://github.com/facebook/react/blob/master/packages/react-dom/src/shared/isCustomComponent.js
```

Comment:
**docs:** could you use the following link instead: https://github.com/facebook/react/blob/v16.6.3/packages/react-dom/src/shared/isCustomComponent.js

This way, it's likely to 404 if it moves around